packages/sources/coinranking/src/adapter.ts

```diff
@@ -1,38 +1,61 @@
-import { Config, ExecuteFactory, ExecuteWithConfig } from '@chainlink/types'
-import { AdapterError, Requester, Validator } from '@chainlink/ea-bootstrap'
-import { DEFAULT_ENDPOINT, makeConfig } from './config'
-import { coin } from './endpoint'
+import { ExecuteWithConfig, ExecuteFactory, Config} from '@chainlink/types'
```

Comment:
Why did an endpoint get moved into the `adapter.ts` code?

Reply:
Resolving in buildSelector PR.